### PR TITLE
fix(*): update for lint error and test failed

### PR DESCRIPTION
**Pull request by mt-rory**
According to the linter error report and unit-test failed report to update codebase.

### CODE REVIEW

**Review by mt-kenny — APPROVED** [review]

File: src/api/__tests__/authorize.test.ts [hunk 1]
```diff
@@ -65,24 +63,20 @@ describe('api', () => {
...
         country,
         locale,
         configs: generateConfigs(),
       });
       const url = `${MY_ACCOUNT_DOMAINS.production}/oauth/authorize?${query}`;
       expect(open).toBeCalledWith(url, '_self');
 
-      expect(mockedStorage.set).toBeCalledTimes(1);
-      expect(mockedStorage.set).toBeCalledWith('codeVerifier', codeVerifier);
+      expect(mockedStorage.set).toBeCalledTimes(0);
```
Comment (mt-max) [inline src/api/__tests__/authorize.test.ts:73]: don't need this

File: src/api/__tests__/authorize.test.ts [hunk 2]
```diff
@@ -105,17 +98,14 @@ describe('api', () => {
...
         state,
         country,
         configs: generateConfigs(),
       });
       const url = `${MY_ACCOUNT_DOMAINS.production}/oauth/authorize?${query}`;
       expect(open).toBeCalledWith(url, '_self');
 
-      expect(mockedStorage.set).toBeCalledTimes(2);
-      expect(mockedStorage.set).toBeCalledWith('codeVerifier', codeVerifier);
+      expect(mockedStorage.set).toBeCalledTimes(0);
```
Comment (mt-max) [inline src/api/__tests__/authorize.test.ts:108]: same

File: src/api/__tests__/onboard.test.ts [hunk 3]
```diff
@@ -94,24 +92,20 @@ describe('api', () => {
...
         country,
         locale,
         configs: generateConfigs({ email }),
       });
       const url = `${MY_ACCOUNT_DOMAINS.production}/onboard?${query}`;
       expect(open).toBeCalledWith(url, '_self');
 
-      expect(mockedStorage.set).toBeCalledTimes(1);
-      expect(mockedStorage.set).toBeCalledWith('codeVerifier', codeVerifier);
+      expect(mockedStorage.set).toBeCalledTimes(0);
```
Comment (mt-max) [inline src/api/__tests__/onboard.test.ts:102]: same

File: src/api/__tests__/onboard.test.ts [hunk 4]
```diff
@@ -135,17 +128,14 @@ describe('api', () => {
...
         state,
         country,
         configs: generateConfigs({ email }),
       });
       const url = `${MY_ACCOUNT_DOMAINS.production}/onboard?${query}`;
       expect(open).toBeCalledWith(url, '_self');
 
-      expect(mockedStorage.set).toBeCalledTimes(2);
-      expect(mockedStorage.set).toBeCalledWith('codeVerifier', codeVerifier);
+      expect(mockedStorage.set).toBeCalledTimes(0);
```
Comment (mt-max) [inline src/api/__tests__/onboard.test.ts:138]: same

**Review by mt-kenny — APPROVED** [review]

**Pull request merged**